packages/combobox/test/combobox.data.test.ts

```diff
@@ -275,20 +322,28 @@ describe('Combobox Data', () => {
         await elementUpdated(test);
         await elementUpdated(el);
 
-        await waitUntil(() => !!el.options?.length);
+        await waitUntil(() => {
+            return !!el.optionEls?.length;
+        });
 
-        expect(el.options).to.deep.equal(options);
+        const processedOptions = el.availableOptions.map(({ id, value }) => ({
+            id,
+            value,
+        }));
+
+        expect(processedOptions).to.deep.equal(options);
     });
-    it('accepts adding through slots', async () => {
+    it('accepts adding through slots', async function () {
+        this.retries(0);
```

Comment:
Do we want to remove this or keep it for now and remove it when we merge to main? If so, we should probably make a note.

Reply:
Good catch; removed!